packages/edit-site/src/components/global-styles/border-panel.js

```diff
@@ -16,80 +16,90 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { useSetting } from '../editor/utils';
+import { getSupportedGlobalStylesPanels, useSetting, useStyle } from './hooks';
```

Comment:
Hopefully not a silly question, but — is the old `useSetting` from `editor/utils` going to be replaced entirely by the new one in `hooks` (and removed) ?

And similarly, are the `getSetting`/`setSetting`/`getStyle`/`setStyle` functions from `global-styles-provider` also going to be replaced entirely from the new ones in `hooks`?

Reply:
Yes, that sounds to be like the plan to replace the context provider and the getters and setters with hooks.

Reply:
I removed a lot of code already, there might be other code that can be removed (maybe some constants in the wordpress/blocks package)